userspace/libsinsp/ifinfo.cpp

```diff
@@ -263,13 +263,13 @@ bool sinsp_network_interfaces::is_ipv4addr_in_local_machine(uint32_t addr, sinsp
 
 				for(const auto it : *clist)
```

Comment:
thinking out loud...

Not sure how often we do stuff like this, but we could consider keying a container map by ip so we don't have to search.

Reply:
Scope creep is strong with this one :P